source/blog/2017-02-16-hyperconverged-ovirt-with-ovn-part1.md

```diff
@@ -0,0 +1,203 @@
+repo_gpgcheck=0
+enabled=1
+enabled_metadata=1
+EOF
+```
+
+Choose one of the three host to be used to run gdeploy and ovirt-hosted-engine setup, we call it the first host.
+On the first host install gdeploy and ovirt-engine-appliance
+```
+yum install -y cockpit-ovirt-dashboard gdeploy ovirt-engine-appliance
+```
+and generate and distribute an ssh key for key based password-less authentication, try it out to ensure that it's working and add each host to know hosts file.
+```
+# generate and distribute ssh key
+ssh-keygen
+ssh-copy-id root@demovm1.localdomain
+ssh-copy-id root@demovm2.localdomain
+ssh-copy-id root@demovm3.localdomain
+# try it on demovm1.localdomain demovm2.localdomain demovm3.localdomain to add ssh keys to known hosts
+```
+On additional hosts simply install cockpit-ovirt-dashboard and gdeploy to fetch all the required dependencies.
+```
+yum install -y cockpit-ovirt-dashboard gdeploy
+```
+On all the three hosts enable and start cockpit and add a firewall rule for it:
+```
+systemctl enable cockpit
+systemctl start cockpit
+firewall-cmd --zone=public --add-port=9090/tcp --permanent
+firewall-cmd --reload
+```
+If you are using VMs as your nodes, you should consider using a virtio-rng device on them to provide a source of entropy since going low on entropy can significantly slow down your demo env.
+```
+yum install -y epel-release
+yum install -y haveged
+systemctl start haveged
+systemctl enable haveged
+yum remove -y epel-release
+```
+If you are using EPEL please take care or removing it since it's going to make the installation of oVirt fail due to a dependency conflict.
+
+## Setup
+
+Now we can point your browser to https://demovm1.localdomain:9090/ to connect to cockpit running on your first host. Connect as root user.
+Point to Virtualization Tab and then Hosted Engine sub tab and choose there 'Hosted Engine with Gluster' as for picture 1.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture01.png")
+As the first step add the hostname of your three hosts on the *storage network*, since we have a single nic on each host I could simply use the FQDN but to make it more clear that's a different thing I’m going to use the host IP addresses as in picture 2.
+![2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png](2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png "2017-02-16-hyperconverged-ovirt-with-ovn-picture02.png")
+On the *Packages* step we can just keep everything as it is since we already installed what's needed.
+On the *Volumes* step it will propose by default to use three volumes but we are going to remove the vmstore one since we just want to build the minimal env for this demo lab so proceed as for picture 3. Please note that gdeploy will propose by default to use replica with arbiter mode to provide more space efficiency than a pure replica 3 mode.
```

Comment:
From oVirt 4.1.1 arbiter will not be enabled by default. If required user has to enable by clicking the check box.